static/app/utils/dates.tsx

```diff
@@ -224,12 +224,12 @@ export function statsPeriodToDays(
   return 0;
 }
 
-export function use24Hours() {
```

Comment:
Cannot call hooks inside "functions". Linter thinks this is a hook because it starts with use*